src/modules/languages/javascript/index.ts

```diff
@@ -3,7 +3,7 @@ import HermitOptions from '../../../utils/lib/HermitOptions';
 import logger from '../../../utils/logger';
 
 export const languageImages = [
-  "node:14",
+  "node:18-slim",
```

Comment:
That was a smart move. That way, the containers will be more lightweight.

Reply:
Exactly